playwright.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    testDir: './tests',
+    timeout: 12000,
```

Comment:
```suggestion
    timeout: 12000,
    fullyParallel: true,
```
Good to make sure that the tests can run and succeed across works, we can opt-out in individual test files